### ISSUE TITLE
Hiding Photos with a Tag

### ISSUE THREAD

**Issue by 19kmunz** [pulsejet/memories#805]
**Is your feature request related to a problem? Please describe.**
When I am going through memories, I tend to want to see everything i have done myself. In reality, there are a lot of other things in my camera roll that will obscure my "Trip to Montreal" or "Random time I meet up with friends in 2015". In particular, lots and lots of memes. If I tagged all of the memes in my nextcloud, and was able to hide them for "on this day" or my main feed, it would be much easier to take a walk down memory lane. People could also use this to hide photos of things that make them uncomfortable but want/need to keep (ex-partners, friends, or family, gross things you sent to your doctor, etc.)

**Describe the solution you'd like**
I want to be able to select tags (either in setting or on the tags page) that will be hidden from view on the On This Day feed and/or Timeline feeds.

**Describe alternatives you've considered**
Memories does have hidden folders, but i do want to keep my existing folder structure as it is. When I am looking at an individual date on disk, i want to see everything for that date in one location. The way I have structure my data is important to me, so I want to use tagging for extra filtering instead of folder structure.

**Comment by eil047** [comment]
I totally agree with this point of view: filtering instead of restructuring storage

**Comment by jacksbox** [comment]
+1

**Comment by Pofilo** [comment]
I thought the "Hide album" feature could help us, but it does not hide any photos.
I prefixed my album with a dot, now in the photo details, the album is not shown.

But yes, being able to hide photos (with tags and/or albums) is a very good idea (rather than moving them in the archive folder).
